### ISSUE TITLE
Unable to export  diagram to draw.io

### ISSUE THREAD

**Issue by madhubysani01** [awslabs/aws-perspective#107]
If your issue relates to the _Discovery Process_, please first follow the steps described in the implementation guide [Debugging the Discovery Component](https://docs.aws.amazon.com/solutions/latest/aws-perspective/appendix-e-debugging-the-discovery-component.html)

---

**Describe the bug**
After generating the diagram,  would like to download as draw.io diagram for future use.  When clicked on the draw.io under export options,  getting following error:
typeerror: Converting circular structure to JSON    --> starting at object with constructor 'Oe'    --- property '0' closes the circle

**To Reproduce**

**Expected behavior**
A description of what you expected to happen.

**Screenshots**

**Browser (please complete the following information):**

- Name [e.g. chrome, firefox]

**Additional context**

**Comment by thebutler12** [comment]
Thanks for raising this. Would it be possible to get an example of the architecture diagram you were trying to export please? It will help us replicate the issue.

**Comment by atcom** [comment]
Hi, 
I have similar issue with perspective with both API and UI.
When I try to open item in draw.io I get the CORS error:
Access to XMLHttpRequest at 'https://xxxxx.execute-api.eu-central-1.amazonaws.com/Prod//resources' from origin 'https://xxxxx.cloudfront.net' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

It happens to all diagrams so it is fairy simple to reproduce.

**Comment by thebutler12** [comment]
Please create a new issue tagged with v1.1.0 if you are still experiencing this issue.
